### PR TITLE
Parallelize bundler

### PR DESCRIPTION
**Pull request by alachaum**

### CODE REVIEW

**Comment by alachaum** [comment]
@ouranos you may be interested in this one 😃

File: 2.3/docker-entrypoint.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ export RACK_ENV=${RACK_ENV:-production}
 export RAILS_ENV=${RAILS_ENV:-production}
 export RAILS_LOG_TO_STDOUT=${RAILS_LOG_TO_STDOUT:-true}
 export GIT_BRANCH=${GIT_BRANCH:-master}
+export BUNDLE_JOBS=${BUNDLER_JOBS:-$(nproc)} # default to number of cores
```
Comment (ouranos) [inline 2.3/docker-entrypoint.sh]: Is this a typo or is it normal? 
`BUNDLE_JOBS=${BUNDLER_JOBS`

**Review by ouranos — APPROVED** [review]
Sweet :+1:

File: 2.3/docker-entrypoint.sh (hunk 1 above)
Comment (alachaum) [reply 2.3/docker-entrypoint.sh]: good pickup - it was a typo. I've fixed it.

**Pull request merged**